bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala

```diff
@@ -59,7 +58,6 @@ trait ItcTable extends Table {
   peer.setRowSelectionAllowed(true)
   peer.setColumnSelectionAllowed(false)
   peer.setSelectionMode(ListSelectionModel.SINGLE_SELECTION)
-  peer.getTableHeader.setReorderingAllowed(false)
```

Comment:
Did you try if the tooltips are ok after you reorder? Sometimes the table indexes get out of whack when you reorder